appveyor.yml

```diff
@@ -31,4 +34,5 @@ install:
 
 artifacts:
  - path: bin
-   name: rpcs3-$(configuration)-$(COMMIT_SHA)
+   name: 'rpcs3-v0.0.1-$(COMMIT_SHA)_$(Date)_win64'
```

Comment:
Makes sense, the reason I did it like that was to prevent people from copying the version until the date and leaving out the SHA when reporting stuff, but it can easily be swapped. 
One could also remove the dashes in the date (so yyyymmdd instead of yyyy-mm-dd) to reduce filename size but I do believe it looks better with dashes.